page-resources.hbs

```diff
@@ -41,14 +41,14 @@
         
     <div id="mission" data-aos="fade-up" class="section-1">
 
-        <div class="text-heading" > <h2 style="margin-top: 100px;">Posters</h2></div>
+        <div class="text-heading text-upper" > <h2> RESOURCES </h2></div>
         <div class="text-body"> {{#page}} {{content}} {{/page}} </div>
         
     </div>
 
     <section class="flex-grid-thirds" style="margin-bottom: 50px">
 
-        <a href="   https://drive.google.com/file/d/1DCkjFweTrxfWZcsn8bDnJhh1nAev0Wmh/view" target="_blank"><div data-aos="fade-up" class="left "> <img class="responsive-poster-size hvr-float"  src="{{asset "images/poster1.png"}}" width="410" height="532">
+        <a href="   https://drive.google.com/file/d/1DCkjFweTrxfWZcsn8bDnJhh1nAev0Wmh/view" target="_blank"><div data-aos="fade-up" class="left "> <img class="responsive-poster-size hvr-float"  src="{{asset "images/poster1.png"}}"  width="410" height="532">

```

Comment:
quite minor but it’s sometimes nice to excise “noise” like this, I assume it’s not a deliberate change?

Reply:
Oops. I was going to add alt text, but decided I'd add it in another pull request.